cmd/examples.go

```diff
@@ -110,10 +108,10 @@ func (e *Example) GetCmd() *cobra.Command {
 				return err
 			}
 			if customSetup {
-				fmt.Println(examples)
+				e.logger.Info(examples)
```

Comment:
Can you provide screenshots of before and after this change? I would like to see which one is better.

Reply:
at the moment it looks like 
<img width="1440" alt="image" src="https://github.com/keploy/keploy/assets/60891544/2878cb0e-0f49-4388-94ab-e4d6499db6a5">


earlier
<img width="1440" alt="image" src="https://github.com/keploy/keploy/assets/60891544/a048fff7-945f-4863-b6d6-2941091df919">


earlier is better